src/mw-utils/services/mw_loading_service.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+angular.module('mwUI.Utils')
+
+/**
+ * @ngdoc service
+ * @name mwUI.Utils.service:Loading
+ *
+ * @description
+ * This service manages loading processes. It can be used e.g. by a httpInterceptor to register a loading process
+ */
+  .service('Loading', function ($timeout) {
```

Comment:
We already have a service called 'Loading' (see relution/common). How are name collisions handled

Reply:
So in fact you moved the loading service into the UI-Kit...

Reply:
Yes with some small modifications, a merge request for the portal will come as soon as this mr is released in a new uikit version